contexts/RefetchContext.tsx

```diff
@@ -28,6 +33,13 @@ export const RefetchProvider: React.FC = ({ children }) => {
 	const marketQuery = useGetFuturesMarket();
 	useGetFuturesPotentialTradeDetails();
 
+	const { network } = Connector.useContainer();
+	const futuresMarketsQuery = useGetFuturesMarkets();
```

Comment:
We are updating these hooks to set data in recoil. Here is an example of how they differ ([setting value](https://github.com/Kwenta/kwenta/blob/fd444c2d24a628ccf50dd04ed04cd4133147108f/queries/futures/useGetFuturesMarket.ts#L81) and [recoil value](https://github.com/Kwenta/kwenta/blob/fd444c2d24a628ccf50dd04ed04cd4133147108f/queries/futures/useGetFuturesMarket.ts#L22))

For the new hooks here (`useGetFuturesMarkets` and `useGetFuturesPositionForMarkets`) we would want these set up in a similar way.